File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.java
class Solution {
    public int solution(int[] num_list) {
        StringBuilder answer1 = new StringBuilder();
        StringBuilder answer2 = new StringBuilder();
        
        for(int i = 0; i < num_list.length; i++){
            if(num_list[i] % 2 == 1){
                answer1.append(num_list[i]);          
            }else{
                answer2.append(num_list[i]);
            }
        }
        return Integer.parseInt(answer1.toString()) + Integer.parseInt(answer2.toString());
    }
}

File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.js
function solution(num_list) {
    let answer1 = '';
    let answer2 = '';

    for(let i = 0; i < num_list.length; i++){
        if(num_list[i] % 2 === 1){
            answer1 += num_list[i].toString();
        }else if(num_list[i] % 2 === 0){
            answer2 += num_list[i].toString();                        
        }   
    }
    return Number(answer1) + Number(answer2);
}